doc/developer-guides/hld/ivshmem-hld.rst

```diff
@@ -127,73 +127,75 @@ MMIO Registers Definition
+community-approved design for shared memory between VMs, following the same
 specification for KVM/QEMU (`Link <https://git.qemu.org/?p=qemu.git;a=blob_plain;f=docs/specs/ivshmem-spec.txt;hb=HEAD>`_).
 
 Following the ACRN threat model, the policy definition for allocation and
 assignment of these regions is controlled by the Service VM, which is part of
 ACRN's Trusted Computing Base (TCB). However, to secure inter-VM communication
-between any userspace applications that harness this channel, applications will
+between any user space applications that harness this channel, applications will
 face more requirements for the confidentiality, integrity, and authenticity of
-shared or transferred data. It is the application development team's
-responsibility to define a threat model and security architecture for the
-application and utilize custom or public libraries accordingly. In this document
-we provide an overview about potential hardening techniques from a userspace
+shared or transferred data. The application development team is
+responsibility for defining a threat model and security architecture for the
+application and utilizing custom or public libraries accordingly. This document
+provides an overview about potential hardening techniques from a user space
 application's perspective. Consider these techniques when defining the security
 architecture and threat model for your application.
 
-.. note:: This is not a definitive guide on all security technologies or how to implement security. We provide general pointers not bounded to a specific OS or use-case.
+.. note:: This document is not a definitive guide on all security technologies
+   or how to implement security. We provide general pointers not bounded to a
+   specific OS or use case.
 
 1. **Secure Feature Configurability**
 
-   - ACRN ensure a minimal control plane for the configuration of the memory region's boundaries and name handles. This is managed only by the Service VM during the creation of the guest VM through the Device Model (DM).
-   - Service VM Admin should refer to the :ref:`usage guide <ivshmem-hld>` for secure configuration flow.
-   - Create different permissions or groups for the ``admin`` role to isolate it from other entities that might have access to the Service VM. For example only admin permissions allow R/W/X on DM binary.
+   - ACRN ensures a minimal control plane for the configuration of the memory region's boundaries and name handles. This capability is managed only by the Service VM during the creation of the guest VM through the Device Model (DM).
+   - Service VM Admin should refer to the :ref:`usage guide <ivshmem-hld>` for the secure configuration flow.
+   - Create different permissions or groups for the ``admin`` role to isolate it from other entities that might have access to the Service VM. For example, only admin permissions allow R/W/X on the DM binary.
    - Reference: :ref:`ivshmem-hld`
```

Comment:
This is another pointer to this very document, I don't think it adds any value here, can we delete it?

Reply:
Done.